### PR TITLE
documentation(EJ2-845801): committed sample changes in the code snippet.

### PR DESCRIPTION
**Pull request by indumathi-murali**

### CODE REVIEW

**Comment by indumathi-murali** [comment]
Tested sample link - 
https://stackblitz.com/edit/nyvca2-zpexyt?file=index.js
https://stackblitz.com/edit/comczr-4mwhbd?file=index.ts

File: ej2-javascript/code-snippet/treemap/how-to/drill-down-cs1/index.js [hunk 1]
```diff
@@ -1,52 +1,87 @@
...
+                groupPath: 'Region',
+                showHeader: false,
+                fill: '#336699',
+                border: { color: 'black', width: 0.5 },
+            },
+        ],
+        loaded: function (args) {
+            var header = document.getElementById('header');
+            var population = 0;
+            for (
```
Comment (sabari-senthamaraikannan) [inline ej2-javascript/code-snippet/treemap/how-to/drill-down-cs1/index.js:40]: for loop must be in single line.

File: ej2-javascript/code-snippet/treemap/how-to/drill-down-cs1/index.js [hunk 2]
```diff
@@ -1,52 +1,87 @@
...
+                var i = 0;
+                i < args.treemap.layout.renderItems[0]['parent'].Continent.length;
+                i++
+            ) {
+                population +=
```
Comment (sabari-senthamaraikannan) [inline ej2-javascript/code-snippet/treemap/how-to/drill-down-cs1/index.js:45]: `+=  +`  - is two `+` operator needed?

File: ej2-javascript/code-snippet/treemap/how-to/drill-down-cs1/index.js [hunk 3]
```diff
@@ -1,52 +1,87 @@
...
+        },
+        drillEnd: function (args) {
+            var header = document.getElementById('header');
+            var layout = document.getElementById(
+                'container_TreeMap_Squarified_Layout'
+            );
+            var population = 0;
+            if (args.treemap.layout.renderItems[0]['isDrilled']) {
+                for (var i = 0; i < args.treemap.layout.renderItems.length; i++) {
+                    population += +args.treemap.layout.renderItems[i]['data'].Population;
```
Comment (sabari-senthamaraikannan) [inline ej2-javascript/code-snippet/treemap/how-to/drill-down-cs1/index.js:59]: `+= +`

File: ej2-javascript/code-snippet/treemap/how-to/drill-down-cs1/index.js [hunk 4]
```diff
@@ -1,52 +1,87 @@
...
+                    layout.children[0].children[1].innerHTML.split(']')[1] +
+                    ' - ' +
+                    population;
+            } else if (args.treemap.layout.renderItems[0]['parent'].Continent) {
+                for (
+                    var i = 0;
+                    i < args.treemap.layout.renderItems[0]['parent'].Continent.length;
+                    i++
+                ) {
+                    population +=
```
Comment (sabari-senthamaraikannan) [inline ej2-javascript/code-snippet/treemap/how-to/drill-down-cs1/index.js:71]: `+= +`

**Review by sabari-senthamaraikannan — APPROVED** [review]

**Pull request merged**